tests/firewall/firewall-04-tls-sni-enforce/test.rules

```diff
@@ -0,0 +1,11 @@
+# allow tcp handshake
+
+pass tcp any any <> any 443 (flow:not_established; sid:1;)
+
+# allow tls
+
+pass tls any any <> any 443 (flow:established; tls.sni; content:"raw.githubusercontent.com"; sid:2;)
```

Comment:
And would you like to add a rule like
`pass tls any any <> any 443 (flow:established; tx.progress:<tls.sni; sid:3;)`

right ?

Reply:
Cool
Do you have a redmine ticket for this improvement ?

SV PR is good like that, and you can make a special SV-only merge
https://github.com/OISF/suricata-verify/pulls?q=is%3Apr+is%3Aopen+review%3Aapproved+label%3A%22tests+pass%22